src/Sylius/Bundle/ProductBundle/spec/EventListener/SelectProductAttributeChoiceRemoveListenerSpec.php

```diff
@@ -43,8 +47,9 @@ function it_removes_select_product_attribute_choices(
 
         $productAttribute->getType()->willReturn(SelectAttributeType::TYPE);
 
-        $entityManager->getUnitOfWork()->willReturn($unitOfWork);
-        $unitOfWork->getEntityChangeSet($productAttribute)->willReturn([
```

Comment:
Need to use Mockery for UnitOfWork mock as Doctrine ORM 2.6 changes `getEntityChangeSet` definition to return a reference instead of a value and it's no supported by Prophecy.

Reply:
To be honest this spec should be rewritten to follow "don't mock what you don't own" rule and behaviour tested by Behat scenarios, but this one is the quickest solution, allows us to install Doctrine ORM 2.6 and fixes the error seen in #9164.

Reply:
Sad but true